client/src/specs/App.spec.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+import App from '../components/App.js';
+
+describe('App', () => {
```

Comment:
I believe snapshot test may not be the best way to write your tests but should work for this project. Reason being snapshot tests will fail if you change something like the h1 to an h2 (Which might happen often during styling of the page). Could work better long term if you test the content inside somehow but overall looks good besides that!